repose-aggregator/src/docs/asciidoc/architecture/filter-chain.adoc

```diff
@@ -0,0 +1,106 @@
+= Filter Chain
```

Comment:
We probably also want a link to this page (perhaps commented our for the time being) in the architecture `index.adoc`?

Reply:
I thought we'd eventually add a page that actually described the overall architecture of repose, and it'd be listed there.

Reply:
Ah, I see. Actually, I guess I kind of did that with the "Servlet Specification" page. The intent was to detail how Repose interacts with the servlet specification. It totally links to the filter chain page.

Even so, I would certainly not be opposed to having this page in the index in addition to being linked to by that page. Personally, I like complete indices that I can reach all pages from.